packages/widgets/src/tabbar.ts

```diff
@@ -807,7 +832,15 @@ export class TabBar<T> extends Widget {
       // Handle the arrow keys to switch tabs.
     } else if (ARROW_KEYS.includes(event.key)) {
       // Create a list of all focusable elements in the tab bar.
-      const focusable: Element[] = [...this.contentNode.children];
+      const focusable: Element[] = [];
+      console.log(focusable);
+
+      if (this.closeLauncherNode) {
+        for (var i = 0; i < this.closeLauncherNode.length; i++) {
+          focusable.push(this.closeLauncherNode[i].parentElement!);
+          focusable.push(this.closeLauncherNode[i]);
```

Comment:
I think it may break UX conventions around tabs to make the close button focusable with the left and right arrows. Do you have any examples of established apps that follow this UX? I noticed that VS Code uses the arrow keys to move between tabs, and once you are focussed on a particular tab, you can press the tab key to focus the close button for that tab (a second press of the tab key takes you away from the tab bar).

What do you think of that behavior?